doc/integrator/headers.rst

```diff
@@ -0,0 +1,26 @@
+.. _integrator_headers:
+
+Headers
+=======
+
+We can set with some configuration some HTTP headers on defferent path.
```

Comment:
What about:

c2cgeoportal produces in its responses HTTP headers with certain default values. You may wish to override the values being written. To achieve this, you can configure a `global _headers`` section in your vars file with the following syntax: